sigstore/_verify.py

```diff
@@ -60,10 +62,15 @@
     "sigstore._store", "fulcio_intermediate.crt.pem"
 )
 
+_OIDC_ISSUER_OID = ObjectIdentifier("1.3.6.1.4.1.57264.1.1")
+
 
 class VerificationResult(BaseModel):
     success: bool
 
+    class Config:
```

Comment:
Hmm, I've had a look at the docs but I'm still why we need this. A comment would help.

Reply:
Yep, I'll add a comment. The short answer is that we need it for the `CertificateVerificationFailedResult` below, since the `exception` member isn't serializable. I wasn't sure whether it made sense to configure it directly, or the entire hierarchy. Thinking about it more, I'll relocate it to just the subclass.